src/FakeItEasy/Core/InterceptedFakeObjectCall.cs

```diff
@@ -21,10 +21,15 @@ internal abstract class InterceptedFakeObjectCall : IInterceptedFakeObjectCall
         public abstract MethodInfo Method { get; }
 
         /// <summary>
-        /// Freezes the call so that it can no longer be modified.
+        /// Gets or sets the return value of the call.
+        /// </summary>
+        public abstract object ReturnValue { get; internal set; }

```

Comment:
I had to choose between adding a `GetReturnValue()` method, or change to a property. The accessor method pair felt to "Javaesque" for my taste...

Reply:
The class is internal, so we could just leave the set public…

Reply:
Good point!